solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -58,6 +58,84 @@ def apply_validation(obs_df, qfilter, handle_func):
         return validated_obs, counts
 
 
+def _resample_event_obs(obs, fx, obs_series):
+    """
+    Resample the event observation.
+
+    Parameters
+    ----------
+    obs : datamodel.Observation
+        The Observation being resampled.
+    fx : datamodel.EventForecast
+        The corresponding Forecast.
+    obs_series : pd.Series
+        Timeseries data of the event observation.
+
+    Returns
+    -------
+    obs_resampled : pandas.Series
+        Timeseries data of the Observation resampled to match the Forecast.
+
+    Raises
+    ------
+    RuntimeError
+        If the Forecast and Observation do not have the same interval length.
+
+    """
+
```

Comment:
maybe want raise exceptions if the fx or obs variable is not event. maybe interval label matching too? But like in  https://github.com/SolarArbiter/solarforecastarbiter-core/pull/387#discussion_r406534834 , I'm not sure what goes here vs. datamodel.

Reply:
now that we have checking in the datamodel, it seems ok to mark this resolved, right?